ibis/expr/operations/udf.py

```diff
@@ -298,10 +298,11 @@ def _make_node(
         func_name = name or fn.__name__
 
         for arg_name, param in inspect.signature(fn).parameters.items():
-            if (raw_dtype := annotations.get(arg_name)) is None:
-                raise exc.MissingParameterAnnotationError(fn, arg_name)
-
-            arg = rlz.ValueOf(dt.dtype(raw_dtype))
+            if (raw_dtype := annotations.get(arg_name)) is not None:
```

Comment:
`pattern = rlz.ValueOf(annotations.get(arg_name))` should be sufficient.

Reply:
Ah, good call!

Reply:
Fixed!

Reply:
Did you push it, I still see the previous diff.

Reply:
Oh, lemme push this up.

Reply:
Ah, it's pushed now!